Quasardb/Query/QdbCell.cs

```diff
@@ -115,8 +122,7 @@ public unsafe string StringValue
                     return null;
                 if (Type != QdbValueType.String)
                     throw new InvalidCastException();
-                var content = new byte[(int)_result.blob_payload.content_size];
-                Marshal.Copy(new IntPtr(_result.blob_payload.content), content, 0, (int)_result.blob_payload.content_size);
+                var content = PayloadToByteArray(_result.string_payload);
                 return System.Text.Encoding.UTF8.GetString(content);
```

Comment:
You still should check for null here. Otherwise https://docs.microsoft.com/en-us/dotnet/api/system.text.utf8encoding.getstring?view=netcore-3.1 will throw.